wordpress-mobile/WordPress-Android
# Issue/16309 compact blogging prompt card

Fixes #16309

This PR ads a compact card to the Create Content bottom sheet. The are some placeholder logic for the answer button, but it is not wider to the button yet. Button toggles between Answer and Answered states for test purposes.

[![Image from Gyazo](https://i.gyazo.com/70a18ce6fa8aae4d54e7a64832a98942.png)](https://gyazo.com/70a18ce6fa8aae4d54e7a64832a98942)

To test:
- Enable BP FF and tap on the + icon on My Site screen.
- Confirm that the compact card is visible at the top of the bottom sheet, and that it looks like the design.
- With the BP FF off, confirm that the compact card is not visible in the bottom sheet.

## Regression Notes
1. Potential unintended areas of impact
- Card is visible when FF is off. 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Atuomated and manual testing.

3. What automated tests I added (or what prevented me from doing so)
- FF test and card click action test.

PR submission checklist:
